### DISCUSSION TITLE
Unique constraint always recreated after `extendedIndexes` enabled

### DISCUSSION THREAD

**Discussion by nasrulfuad** [prisma/prisma#11981] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/39388505/155148472-d2f8cc21-5eb5-485e-a1cc-d2b443812283.png)

Anyone know why, after `extendedIndexes` enabled, I always got these warning every I updated my schema?

**Comment by nurul3101** [comment, marked as answer]
Hey @nasrulfuad 👋🏻 ,
Thank you for reporting this issue. We can confirm that we can reproduce this issue. 
Here is the GitHub Issue for the same:
- https://github.com/prisma/prisma/issues/11631

This will be fixed in the upcoming Prisma releases.
As a workaround, for now, I suggest you run `npx prisma migrate dev --create-only` and then remove the lines that add the unique index from the generated migration file. Please note that this needs to be done every time new migration is created.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
